config/default.yml

```diff
@@ -23,6 +23,10 @@ Chef/AttributeKeys:
   - strings
   - symbols
 
+Chef/ServiceResource:
```

Comment:
I'm not crazy about this name, as it doesn't say _what_ about the Service Resource is wrong, but I'm having a super hard time coming up a succinct name for manipulating service state.